README.md

```diff
@@ -1,84 +1,62 @@
-[![Netlify Status](https://api.netlify.com/api/v1/badges/819c20ba-81db-4dae-83f4-a1dbe82eba5f/deploy-status)](https://app.netlify.com/sites/xenodochial-bohr-1c4dfd/deploys)
-
-<h1 align="center">
-  Support Local Akron
-</h1>
+# Support Local Akron
 
 ## Background
+
 Sitting at home during the COVID-19 pandemic my wife Kirsten and I were trying to figure out how we could best support our city's local artists, musicians, and businesses. It occured to us that we weren't going to be able to answer that question ourselves, so we built this as a way to allow them to tell us, on their terms, how we can support them during this unconventional time.
 
 If you think this will help your city then please clone, fork, download, or copy+paste this codebase into your own repository and start your own #SupportLocal application. No permission required. We're all in this together.
 
 ## Known Issues
+
 You would be amazing if you submitted a PR to fix any of these!
+
 - The photo uploader isn't working on the "Get Listed" form.
-- The sort does not persist when you filter from one category to the next. I'm not sure if this is really an "issue" as I accounted for it in the UI, but it still might be weird.
 - There are no tests.
+
 ### Things That Aren't Issues But Annoy Me
+
 - The CSS is a one-file disaster and doesn't use a pre-processor.
-- The React components are split between class-based and functional. They should all be functional to take advantage of hooks.
 
 ## Supporting Services/Technologies
-- Built with [GatsbyJS](https://www.gatsbyjs.org/) v2.19 (CLI v2.10)
-- Server-side functionality written in [NodeJS](https://nodejs.org/) v12.11.1
-- UI Components made with [ReactJS](https://reactjs.org/) v16.13
-- CMS is [Contentful](https://www.contentful.com/)
-- Data requests made using [GraphQL](https://graphql.org/)
-- CI/CD and hosting through [Netlify](https://www.netlify.com/)
-- Monitoring through [Google Analytics](https://analytics.google.com/)
+
+- Built with [NextJS 14](https://nextjs.org/) using the app router
```

Comment:
Thanks for updating this! With any project, but especially open source, you want the barrier to entry as low as possible. A great README goes a long way.